### PR TITLE
Fixing Baker Missing Stuff

### PR DESCRIPTION
**Pull request by angrycuban13**

### CODE REVIEW

File: whisparr.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- [<i class="fas fa-book"></i>&emsp;API Docs](https://whisparr.video/docs/api/#/)
+{.links-list}
+
+> Note that the API Docs are now auto-generated in the build pipeline for the nightly (GitHub develop) branch {.is-info}
+
+# Support
+
+Still having issues? feel free to stop by one of our support sites for extra help
+
+- [<i class="fab fa-discord"></i>&emsp;Discord *Official Whisparr Discord*](https://whisparr.video/discord)
```
Comment (bakerboy448) [inline whisparr.md:54]: not .video

File: whisparr.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- [<i class="fab fa-reddit"></i>&emsp;Reddit *Official Whisparr Subreddit*](https://reddit.com/r/whisparr)
+{.links-list}
+
+# Links
+
+- [whisparr.video *Website*](https://whisparr.video)
```
Comment (bakerboy448) [inline whisparr.md:60]: fix

File: whisparr.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- [Donate *Donations*](https://whisparr.video/donate)
```
Comment (bakerboy448) [inline whisparr.md:61]: better find/replacing

File: whisparr/activity.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+![history2.png](/assets/whisparr/history2.png)
+
+On `Grabbed` statuses, you can click on the `i` icon on the right to see more details about the download (what indexer it came from, the URL of the grab, the age of the upload, etc.). You can also mark this item as failed, to initiate a removal, blocklist, and re-search of the item.
+
+![history4.png](/assets/whisparr/history4.png)
+
+# Blocklist
+
+> Blocklist is formerly known as 'Blacklist' {.is-info}
```
Comment (bakerboy448) [inline whisparr/activity.md:58]: remove

File: whisparr/installation.md [hunk 5]
```diff
@@ -0,0 +1,792 @@
...
+
+- [Table of Contents](#table-of-contents)
+- [Windows](#windows)
+- [MacOS (OSX)](#macos-osx)
+- [Linux](#linux)
+  - [Debian / Ubuntu](#debian-ubuntu)
+    - [Easy Install](#easy-install)
+    - [Debian / Ubuntu Hands on Install](#debian-ubuntu-hands-on-install)
+    - [Uninstall](#uninstall)
+- [FreeBSD](#freebsd)
```
Comment (bakerboy448) [inline whisparr/installation.md:21]: lies

File: whisparr/installation.md [hunk 6]
```diff
@@ -0,0 +1,792 @@
...
+
+### Easy Install
+
+For the Debian / Ubuntu / Raspbian beginners there isn't an Apt Repository or Deb package.
+
+If you want an easy life, follow this community provided and maintained `Easy Install` script for a base Debian (Raspbian / Raspberry Pi OS) / Ubuntu install.
+
+**For the official installation instructions that are 'Hands on' follow the [Debian / Ubuntu Hands on Install](#debian-ubuntu-hands-on-install) steps further below.**
+
+[Please see the \*Arr Community Installation Script](/install-script)
```
Comment (bakerboy448) [inline whisparr/installation.md:103]: does not support whisparr

File: whisparr/installation.md [hunk 7]
```diff
@@ -0,0 +1,792 @@
...
+> By continuing below, you acknowledge that you have read and met the above requirements. {.is-warning}
+
+- Download the correct binaries for your architecture.
+  - You can determine your architecture with `dpkg --print-architecture`
+    - AMD64 use `arch=x64`
+    - ARM, armf, and armh use `arch=arm`
+    - ARM64 use `arch=arm64`
+
+```shell
+wget --content-disposition 'http://whisparr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore&arch=x64'
```
Comment (bakerboy448) [inline whisparr/installation.md:140]: nightly

File: whisparr/system.md [hunk 8]
```diff
@@ -0,0 +1,572 @@
...
+---
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Status](#status)
+  - [Health](#health)
+    - [System Warnings](#system-warnings)
+      - [Branch is not a valid release branch](#branch-is-not-a-valid-release-branch)
+      - [Update to .NET version](#update-to-net-version)
```
Comment (bakerboy448) [inline whisparr/system.md:18]: drop this

File: whisparr/system.md [hunk 9]
```diff
@@ -0,0 +1,572 @@
...
+        - [Fixing Docker installs](#fixing-docker-installs)
+        - [Fixing FreeBSD installs](#fixing-freebsd-installs)
+        - [Fixing Standalone installs](#fixing-standalone-installs)
+      - [Currently installed mono version is old and unsupported](#currently-installed-mono-version-is-old-and-unsupported)
```
Comment (bakerboy448) [inline whisparr/system.md:22]: bye bye mono

File: whisparr/system.md [hunk 10]
```diff
@@ -0,0 +1,572 @@
...
+      - [Branch is for a previous version](#branch-is-for-a-previous-version)
+      - [Could not connect to signalR](#could-not-connect-to-signalr)
+        - [Nginx](#nginx)
+        - [Apache2](#apache2)
+        - [Caddy](#caddy)
+      - [Failed to resolve the IP Address for the Configured Proxy Host](#failed-to-resolve-the-ip-address-for-the-configured-proxy-host)
+      - [Proxy Failed Test](#proxy-failed-test)
+      - [System Time is off by more than 1 day](#system-time-is-off-by-more-than-1-day)
+      - [MediaInfo Library Could not be Loaded](#mediainfo-library-could-not-be-loaded)
+      - [PTP Indexer Settings Out of Date](#ptp-indexer-settings-out-of-date)
```
Comment (bakerboy448) [inline whisparr/system.md:37]: drop

File: whisparr/system.md [hunk 11]
```diff
@@ -0,0 +1,572 @@
...
+      - [Updating will not be possible to prevent deleting AppData on Update](#updating-will-not-be-possible-to-prevent-deleting-appdata-on-update)
+      - [Branch is for a previous version](#branch-is-for-a-previous-version)
+      - [Could not connect to signalR](#could-not-connect-to-signalr)
+        - [Nginx](#nginx)
+        - [Apache2](#apache2)
+        - [Caddy](#caddy)
+      - [Failed to resolve the IP Address for the Configured Proxy Host](#failed-to-resolve-the-ip-address-for-the-configured-proxy-host)
+      - [Proxy Failed Test](#proxy-failed-test)
+      - [System Time is off by more than 1 day](#system-time-is-off-by-more-than-1-day)
+      - [MediaInfo Library Could not be Loaded](#mediainfo-library-could-not-be-loaded)
```
Comment (bakerboy448) [inline whisparr/system.md:36]: drop

File: whisparr/faq.md [hunk 12]
```diff
@@ -0,0 +1,581 @@
...
+  - [How does Whisparr handle foreign movies or foreign titles?](#how-does-whisparr-handle-foreign-movies-or-foreign-titles)
+  - [How does Whisparr handle "multi" in names?](#how-does-whisparr-handle-multi-in-names)
+  - [Help, Movie Added, But Not Searched](#help-movie-added-but-not-searched)
+  - [Jackett's /all Endpoint](#jacketts-all-endpoint)
+  - [Why are there two files? | Why is there a file left in downloads?](#why-are-there-two-files-why-is-there-a-file-left-in-downloads)
+  - [Why doesn't Whisparr work behind a reverse proxy](#why-doesnt-whisparr-work-behind-a-reverse-proxy)
+
+# Whisparr Basics
+
+## How does Whisparr work?
```
Comment (bakerboy448) [inline whisparr/faq.md:78]: fix

File: whisparr/faq.md [hunk 13]
```diff
@@ -0,0 +1,581 @@
...
+- Whisparr enforces a minimum of 10 minutes on the RSS Sync interval and a maximum of 2 hours. 15 minutes is the minimum recommended by most indexers, though some do allow lower intervals and 2 hours ensures Whisparr is checking frequently enough to not miss a release (even though it can page through the RSS feed on many indexers to help with that). Some indexers allow clients to perform an RSS sync more frequently than 10 minutes, in those scenarios we recommend using Whisparr's Release-Push API endpoint along with an IRC announce channel to push releases to Whisparr for processing which can happen in near real time and with less overhead on the indexer and Whisparr as Whisparr doesn’t need to request the RSS feed too frequently and process the same releases over and over.
+
+## How does Whisparr find movies?
```
Comment (bakerboy448) [inline whisparr/faq.md:83]: fix

File: whisparr/faq.md [hunk 14]
```diff
@@ -0,0 +1,581 @@
...
+- You can trigger a list refresh manually, script it and trigger it via the API, add the movies to Whisparr, use Ombi, Petio, Overseer, or any similar app that adds them right away
+
+- This restriction is to not have our server get killed by people updating lists every 10 minutes.
+
+- This interval can be configured in [Settings => Lists](/whisparr/settings#lists) for between 6-24 hours. The default is 24 hours.
+
+## Can all my movie files be stored in one folder?
+
+- No and the reason is that Whisparr is a fork of [Sonarr](/sonarr), where every show has a folder. This limitation is a known pain point for many users and will maybe come in a future version. Please note that it is not a simple change and effectively requires an entire rewrite of the backend.
+- The [Custom Folder GitHub Issue](https://github.com/Whisparr/Whisparr/issues/153) technically covers this request, but it is no guarantee that all movie files in one folder will be implemented at that time.
```
Comment (bakerboy448) [inline whisparr/faq.md:145]: fix

File: whisparr/faq.md [hunk 15]
```diff
@@ -0,0 +1,581 @@
...
+## How do I update Whisparr?
+
+- Go to Settings and then the General tab and show advanced settings (use the toggle by the save button).
+
+1. Under the Updates section change the branch name to `master` or `develop`
+1. Save
+
+*This will not install the bits from that branch immediately, it will happen during the next update.*
+
+- `master` - ![Current Master/Stable](https://img.shields.io/badge/dynamic/json?color=f5f5f5&style=flat-square&label=&query=%24.version&url=https://raw.githubusercontent.com/hotio/whisparr/release/VERSION.json) - (Default/Stable): It has been tested by users on the develop and nightly branches and it’s not known to have any major issues. This version will receive updates approximately monthly. On GitHub, this is the `master` branch.
```
Comment (bakerboy448) [inline whisparr/faq.md:167]: copy language from prowlarr

File: whisparr/faq.md [hunk 16]
```diff
@@ -0,0 +1,581 @@
...
+
+- `develop` - ![Current Develop/Beta](https://img.shields.io/badge/dynamic/json?color=f5f5f5&style=flat-square&label=&query=%24.version&url=https://raw.githubusercontent.com/hotio/whisparr/testing/VERSION.json) - (Beta): This is the testing edge. Released after tested in nightly to ensure no immediate issues. New features and bug fixes released here first. This version will receive updates either weekly or biweekly depending on development.
```
Comment (bakerboy448) [inline whisparr/faq.md:169]: see what old readarr language was

File: whisparr/faq.md [hunk 17]
```diff
@@ -0,0 +1,581 @@
...
+- See the entry below
+
+## Can I switch between branches?
+
+> You can (almost) always increase your risk.{.is-info}
+
+- See below or otherwise check with the development team to see if you can switch from `nightly` to `master`; `nightly` to `develop`; or `develop` to `master` for your given build.
+- Failure to follow these instructions may result in your Whisparr becoming unusable or throwing errors. You have been warned
+  - The most common error is something like `Error parsing column 45 (Language=31 - Int64)` or other similar database errors around missing columns or tables.
+- Master
```
Comment (bakerboy448) [inline whisparr/faq.md:212]: mirror readarr faq for this

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
rookie moves

**Comment by bakerboy448** [comment]
also conflicted

File: whisparr/activity.md (hunk 4 above)
Comment (angrycuban13) [reply whisparr/activity.md:58]: Radarr needs to be updated then.

File: whisparr/faq.md (hunk 12 above)
Comment (angrycuban13) [reply whisparr/faq.md:78]: Fix to?

**Pull request merged**